Author: Luis Ibanez <luis.ibanez@kitware.com>
Date: Sat, 11 Feb 2012 21:04:05 -0500
Description: Make sure the build will not stop to early

--- fis-gtm-5.4-002B.orig/sr_unix/comlist.mk
+++ fis-gtm-5.4-002B/sr_unix/comlist.mk
@@ -45,7 +45,7 @@
 
 CURDIR=$(shell pwd)
 
-ifeq ($(MAKELEVEL),0)
+ifeq ($(MAKELEVEL),2)
 # the first-level make invocation - rules to create & clean directories
 # and build utilities selectively.
 
